planner/core/planbuilder.go

```diff
@@ -1068,6 +1068,10 @@ func getPossibleAccessPaths(ctx sessionctx.Context, tableHints *tableHintInfo, i
 				ctx.GetSessionVars().StmtCtx.AppendWarning(err)
 				continue
 			}
+			if path.Index.Global {
```

Comment:
When should we skip the global index and would we not skip it later? @tiancaiamao @qw4990

Reply:
global index is not GA feature, I think the progress of completeness is about 40%
It's even not coming to the testing stage yet, so I'm sure if we have more tests, there will be a lot of bugs.

This change looks good to me, it fix a bug (with test to prevent regression) and not adding much complexity to the code base.

If we continue the developing of this feature, and make some exhaustive testing, we will get back to handle it here.